### PR TITLE
Update rule.md

### PR DESCRIPTION
**Pull request by bahjatalaadelssw**
Adding description to Good example image of @ mention formatting in PBIs

### CODE REVIEW

File: rules/when-you-use-mentions-in-a-pbi/rule.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The benefits of using comments are:
 
 
 
-When someone (especially the PO) asks you to fix a PBI, mention that person in the PBI comments so he knows when it’s fixed.
+When someone (especially the PO) asks you to fix a PBI, mention that person in the PBI comments so they knows when it’s fixed.
```
Comment (chrisschultzssw) [inline rules/when-you-use-mentions-in-a-pbi/rule.md:42]: Good change - "knows" should now be changed to "know".

**Review by chrisschultzssw — CHANGES_REQUESTED** [review]

File: rules/when-you-use-mentions-in-a-pbi/rule.md (hunk 1 above)
Comment (bahjatalaadelssw) [reply rules/when-you-use-mentions-in-a-pbi/rule.md:42]: Good catch. Thanks!

**Review by UlyssesMaclaren — APPROVED** [review]

**Review by bahjatalaadelssw — COMMENTED** [review]
Changes made and clarified

**Review by chrisschultzssw — APPROVED** [review]
⭐

**Pull request merged**